sklearn/cross_validation.py

```diff
@@ -524,8 +524,10 @@ class StratifiedKFold(_BaseKFold):
 
     Notes
     -----
-    All the folds have size trunc(n_samples / n_folds), the last one has the
-    complementary.
+    For each class `c`, the first `n_samples_c % n_splits` folds have
+    `n_samples_c // n_splits + 1` samples of class `c`, and other folds have
```

Comment:
why the +1?

Reply:
that doesn't seem to reflect the output that you get in your issue, right?